docs/content/dagster-cloud/deployment/agents/amazon-ecs/configuration-reference.mdx

```diff
@@ -0,0 +1,190 @@
+    be key-value dictionaries.
+  </ReferenceTableItem>
+</ReferenceTable>
+
+If you're using the Dagster Cloud Github action, the `container_context` key can also be set for each location in your `dagster_cloud.yaml` file, using the same format.
+
+---
+
+### Per-job configuration: Resource limits
+
+You can use job tags to customize the CPU and Memory of every run for that job:
+
+```py
+from dagster import job, op
+
+@op()
+def my_op(context):
+  context.log.info('running')
+
+@job(
+  tags = {
+    "ecs/cpu": "256",
+    "ecs/memory": "512",
+  }
+)
+def my_job():
+  my_op()
+```
+
+[Fargate tasks only support certain combinations of CPU and Memory.](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html)
+
+## Per-deployment configuration
+
+This section describes the properties of the `dagster.yaml` configuration file used by Amazon ECS agents. Typically, this file is created by the Cloudformation template that deploys the agent, and can be found with the agent task definition's command. To change these properties, edit the cloudformation template and redeploy the Cloudformation stack.
+
+```yaml
+instance_class:
+  module: dagster_cloud
+  class: DagsterCloudAgentInstance
+
+dagster_cloud_api:
+  agent_token:
+    env: DAGSTER_CLOUD_AGENT_TOKEN
+  deployment: <Deployment>
+
+user_code_launcher:
+  module: dagster_cloud.workspace.ecs
+  class: EcsUserCodeLauncher
+  config:
+    cluster: <Cluster Name>
+    subnets:
+      - <Subnet Id 1>
+      - <Subnet Id 2>
+    security_group_ids:
+      - <Security Group ID>
+    service_discovery_namespace_id: <Service Discovery Namespace Id>
+    execution_role_arn: <Task Execution Role Arn>
+    task_role_arn: <Task Role Arn>
+    log_group: <Log Group Name>
+    launch_type: <"FARGATE"|"EC2">
+    server_process_startup_timeout: <Timeout in seconds>
+```
+
+### dagster_cloud_api properties
+
+<ReferenceTable>
+  <ReferenceTableItem propertyName="deployment">
+    The name of the Dagster Cloud deployment associated with the agent.
+  </ReferenceTableItem>
+</ReferenceTable>
+
+### user_code_launcher properties
+
+<ReferenceTable>
+  <ReferenceTableItem
+  propertyName="config.cluster"
+  >
+  The name of an <a href="https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create_cluster.html" target="_blank">ECS cluster</a> with a <a href="https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cluster-capacity-providers.html" target="_blank">Fargate or EC2 capacity provider</a>.
+  </ReferenceTableItem>
+  <ReferenceTableItem
+  propertyName="config.launch_type"
+  >
+  An <a href="https://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_types.html" target="blank">ECS launch type</a> to use for your launched ECS tasks (currently FARGATE and EC2 are supported). Using an EC2 launch type will require you to have an EC2 capacity provider installed and will require more operational overhead to run the agent.
```

Comment:
I suggest formatting this description like this to make it easier to read:

```suggestion
  An <a href="https://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_types.html" target="blank">ECS launch type</a> to use for your launched ECS tasks. The following are currently supported:
<ul>
<li><code>FARGATE</code></li>
<li><code>EC2</code> - <strong>Note:</strong> Using this launch type requires you to have an EC2 capacity provider installed and additional operational overhead to run the agent.</li>
</ul> 
```

I put `FARGATE` and `EC2` as code because I'm under the assumption that the value must be provided exactly that way (uppercase).